src/auspex/analysis/fits.py

```diff
@@ -1,9 +1,94 @@
 from scipy.optimize import curve_fit
 import numpy as np
 from numpy.fft import fft
+from scipy.linalg import svd, eig, inv, pinv
 from enum import Enum
 
-import matplotlib.pyplot as plt
+def hilbert(signal):
+    # construct the Hilbert transform of the signal via the FFT
+    # in essense, we just want to set negative frequency components to zero
+    spectrum = np.fft.fft(signal)
+    n = len(signal)
+    midpoint = int(np.ceil(n/2))
+
+    kernel = np.zeros(n)
+    kernel[0] = 1
+    if n%2 == 0:
+        kernel[midpoint] = 1
+    kernel[1:midpoint] = 2
+    return np.fft.ifft(kernel * spectrum)
+
+def KT_estimation(data, times, order):
+    # Find the hilbert transform
+    analytic_signal = hilbert(data)
+    time_step = times[1]-times[0]
+
+    # Create the Hankel matrix
+    N = len(analytic_signal)
+    K = order
+    M = (N//2)-1
+    L = N-M+1
+    H = np.zeros((L, M), dtype=np.complex128)
+    for ct in range(M):
+        H[:,ct] = analytic_signal[ct:ct+L]
+
+    #Try and seperate the signal and noise subspace via the svd
+    U,S,V = svd(H, False)
+    V = V.T # not transposed/conjugated in numpy svd
```

Comment:
you're  just transposing again on line 46

Reply:
True.